src/libcollections/vec_deque.rs

```diff
@@ -2401,4 +2438,24 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn test_from_vec() {
+        let mut tester = VecDeque::new();
+        tester.push_back(2);
+        tester.push_back(3);
+        tester.push_front(1);
+        assert_eq!(tester, VecDeque::from(vec![1, 2, 3]));
+    }
+
+    #[test]
+    fn test_into_vec() {
```

Comment:
This could also benefit from some other test cases to ensure that the use of `wrap_copy` is correct, with varying item layouts (contiguous, wrapping, etc).